src/depackers/stb_vorbis.c

```diff
@@ -734,19 +1098,15 @@ static int compute_codewords(Codebook *c, uint8 *len, int n, uint32 *values)
       // more than one free leaf at a given level, isn't totally
       // trivial to prove, but it seems true and the assert never
       // fires, so!
-
-      /* Sanity check */
-      if (z >= 32) {
-        return FALSE;
-      }
-
       while (z > 0 && !available[z]) --z;
-      if (z == 0) { /*assert(0);*/ return FALSE; }
+      if (z == 0) { return FALSE; }
       res = available[z];
+      assert(z >= 0 && z < 32);
```

Comment:
clobbered bugfix, (Also, asserts are a really poor excuse for bounds checks on arbitrary data.)